src/components/BooksForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+const BooksForm = () => {
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
```

Comment:
- [ ] To get rid of this linter issue (label has associated control), you need to wrap the input element inside the label with the `htmlFor` attribute like this:

```html
<label htmlFor="title">
  Title
  <input id="title" type="text" name="title" />
</label>
```

Read more about this [error](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md)